routes/api_v2.php

```diff
@@ -330,7 +329,6 @@
     });
 
     Route::prefix('sites')->group(function () {
-        Route::get('/', AdminIndexSitesController::class);
```

Comment:
This should also be removing the `{project}/sites` route, and comment on `ViewEntityController` should be updated to note that Site is also no longer being fetched from v2. (and associated documentation for those routes should be removed)